coverage/summary.py

```diff
@@ -104,10 +104,21 @@ def report(self, morfs, outfile=None):
 
         # Sort the lines and write them out.
         if getattr(self.config, 'sort', None):
-            position = column_order.get(self.config.sort.lower())
+            _sort = self.config.sort.lower()
+            if _sort[0] == '-':
+                reverse = True
+                sort_option = _sort[1:]
+            elif _sort[0] == '+':
+                reverse = False
+                sort_option = _sort[1:]
+            else:
+                reverse = False
+                sort_option = _sort
```

Comment:
We can tighten up this code a bit:
```suggestion
            sort_option = self.config.sort.lower()
            reverse = False
            if sort_option[0] == '-':
                reverse = True
                sort_option = sort_option[1:]
            elif sort_option[0] == '+':
                sort_option = sort_option[1:]
```

Reply:
yeah, this is much better.